src/GraphMsg.chpl

```diff
@@ -0,0 +1,4510 @@
+      var sDirected=directed:string;
+      var sWeighted=weighted:string;
+
+      var srcNameR, dstNameR, startNameR, neiNameR:string;
+      if (directed!=0) {//for directed graph
+          if (weighted!=0) {// for weighted graph
+              repMsg =  sNv + '+ ' + sNe + '+ ' + sDirected + '+ ' + sWeighted +
+                    '+created ' + st.attrib(srcName)   + '+created ' + st.attrib(dstName) +
+                    '+created ' + st.attrib(startName) + '+created ' + st.attrib(neiName) +
+                    '+created ' + st.attrib(vwName)    + '+created ' + st.attrib(ewName);
+          } else {// for unweighted graph
+              repMsg =  sNv + '+ ' + sNe + '+ ' + sDirected + '+ ' + sWeighted +
+                    '+created ' + st.attrib(srcName)   + '+created ' + st.attrib(dstName) +
+                    '+created ' + st.attrib(startName) + '+created ' + st.attrib(neiName) ;
+
+          }
+      } else {//for undirected graph
+
+          srcNameR = st.nextName();
+          dstNameR = st.nextName();
+          startNameR = st.nextName();
+          neiNameR = st.nextName();
+          var srcEntryR = new shared SymEntry(srcR);
+          var dstEntryR = new shared SymEntry(dstR);
+          var startEntryR = new shared SymEntry(start_iR);
+          var neiEntryR = new shared SymEntry(neighbourR);
+          st.addEntry(srcNameR, srcEntryR);
+          st.addEntry(dstNameR, dstEntryR);
+          st.addEntry(startNameR, startEntryR);
+          st.addEntry(neiNameR, neiEntryR);
+          if (weighted!=0) {// for weighted graph
+              repMsg =  sNv + '+ ' + sNe + '+ ' + sDirected + ' +' + sWeighted +
+                    '+created ' + st.attrib(srcName)   + '+created ' + st.attrib(dstName) +
+                    '+created ' + st.attrib(startName) + '+created ' + st.attrib(neiName) +
+                    '+created ' + st.attrib(srcNameR)   + '+created ' + st.attrib(dstNameR) +
+                    '+created ' + st.attrib(startNameR) + '+created ' + st.attrib(neiNameR) +
+                    '+created ' + st.attrib(vwName)    + '+created ' + st.attrib(ewName);
+          } else {// for unweighted graph
+              repMsg =  sNv + '+ ' + sNe + '+ ' + sDirected + ' +' + sWeighted +
+                    '+created ' + st.attrib(srcName)   + '+created ' + st.attrib(dstName) +
+                    '+created ' + st.attrib(startName) + '+created ' + st.attrib(neiName) +
+                    '+created ' + st.attrib(srcNameR)   + '+created ' + st.attrib(dstNameR) +
+                    '+created ' + st.attrib(startNameR) + '+created ' + st.attrib(neiNameR) ;
+          }
+
+      }
+      timer.stop();
+      writeln("$$$$$$$$$$$$  $$$$$$$$$$$$$$$$$$$$$$$");
+      writeln("$$$$$$$$$$$$  $$$$$$$$$$$$$$$$$$$$$$$");
+      writeln("$$$$$$$$$$$$Sorting Edges takes ", timer.elapsed()," $$$$$$$$$$$$$$$$$$$$$$$");
+      writeln("$$$$$$$$$$$$  $$$$$$$$$$$$$$$$$$$$$$$");
+      writeln("$$$$$$$$$$$$  $$$$$$$$$$$$$$$$$$$$$$$");
```

Comment:
These writeln's should be reduced and turned into a Logger statement.

Reply:
OK. I will handle the debugging information as Logger statement.